tests/conftest.py

```diff
@@ -5,9 +5,15 @@
 import pythoncom
 import nitsm.codemoduleapi
 
-_standalone_tsm_context_tlb = win32com.client.selecttlb.FindTlbsWithDescription(
-    "NI TestStand Semiconductor Module Standalone Semiconductor Module Context"
-)[0]
+try:
+    _standalone_tsm_context_tlb = win32com.client.selecttlb.FindTlbsWithDescription(
+        "NI TestStand Semiconductor Module Standalone Semiconductor Module Context"
+    )[0]
+except IndexError:
+    raise IndexError(
```

Comment:
Thoughts on a different error type for a missing component?
I was thinking perhaps it would be more appropriate to raise a [RuntimeError](https://docs.python.org/3.6/library/exceptions.html#RuntimeError).  The other built-in exceptions don't seem to have a good match for what is happening here.